tuning/src/main/java/frc/robot/XboxRaw.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot;
+
+// Constants for the raw button and axis values of the XBox controller
+public enum XboxRaw {
+  A(1),
```

Comment:
If this is the file from robotlib it is deprecated. I thought I already removed it from robotlib. It needs to go away.